packages/react/src/components/Layout/Grid-story.js

```diff
@@ -0,0 +1,256 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import './story.scss';
+
+import { storiesOf } from '@storybook/react';
+import React, { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import {
+  unstable_Grid as Grid,
+  unstable_Row as Row,
+  unstable_Column as Column,
+} from '../Layout';
+
+function DemoFullPage({ children }) {
+  const [portalNode, setPortalNode] = useState(null);
+
+  useEffect(() => {
+    const node = document.createElement('div');
+    document.body.appendChild(node);
+
+    setPortalNode(node);
+
+    return () => {
+      document.body.removeChild(node);
+    };
+  }, []);
+
+  return (
+    portalNode &&
+    ReactDOM.createPortal(<div className="example">{children}</div>, portalNode)
+  );
+}
+
+function DemoContent({ children }) {
+  return (
+    <div className="outside">
+      <div className="inside">{children}</div>
+    </div>
+  );
+}
+
+storiesOf('Layout/Grid', module)
+  .addDecorator(story => <DemoFullPage>{story()}</DemoFullPage>)
+  .add('auto-columns', () => (
+    <Grid>
+      <Row>
+        <Column>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+        <Column>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+        <Column>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+        <Column>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+      </Row>
+    </Grid>
+  ))
+  .add('responsive grid', () => (
+    <Grid>
+      <Row>
+        <Column span={[1, 4, 8]}>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+        <Column span={[1, 2, 2]}>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+        <Column span={[1, 1, 1]}>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+        <Column span={[1, 1, 1]}>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+      </Row>
+    </Grid>
+  ))
+  .add('responsive grid - object syntax', () => (
+    <Grid>
+      <Row>
+        <Column span={{ sm: 1, md: 4, lg: 8 }}>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+        <Column span={{ sm: 1, md: 2, lg: 2 }}>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+        <Column span={{ sm: 1, md: 1, lg: 1 }}>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+        <Column span={{ sm: 1, md: 1, lg: 1 }}>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+      </Row>
+    </Grid>
+  ))
+  .add('offset', () => (
+    <Grid>
+      <Row>
+        <Column offset={[3]} span={[1]}>
+          <DemoContent>Offset 3</DemoContent>
+        </Column>
+        <Column offset={[2]} span={[2]}>
+          <DemoContent>Offset 3</DemoContent>
```

Comment:
```suggestion
          <DemoContent>Offset 2</DemoContent>
```

and same with the other columns below?

Reply:
@emyarod I'm definitely torn, when folks are viewing the storybooks often they want something they can copy-paste and knobs can impede that use-case through a layer of indirection.

Reply:
I definitely don't like knobs when I'm looking to our React examples to try and copy some example code.

Reply:
the demo content is meant to reflect the offset value though right

Reply:
Oh agreed, I think it's 100% a copy issue